agent/util-scripts/pbench-make-result-tb

```diff
@@ -162,6 +162,14 @@ result_size=$(du -sb ${pbench_run_name} | awk '{print $1}')
 debug_log "preparing to tar up ${result_size} bytes of data from ${full_result_dir}"
 printf -- "$result_size" | pbench-add-metalog-option ${mdlog} run raw_size
 
+times=$(date -u "+%Y-%m-%dT%H:%M:%S-%Z")
+if [[ -z "$_PBENCH_BENCH_TESTS" ]]; then
+	# use ns in the timestamp
+	printf "${times}" | pbench-add-metalog-option ${mdlog} pbench timestamp
+else
+	printf "1900-01-01T00:00:00.000000" | pbench-add-metalog-option ${mdlog} pbench timestamp
+fi
+
```

Comment:
No, we don't want to put *any* knowledge about unit tests in the scripts themselves. All the scripts source the base script: the base script is the *only one* that knows about unit tests, just so that no other script needs to know. All that *this* script needs to do is to use the `timestamp` function that is defined in the base script: that function will return a fake timestamp if running in the unit test regime or a real one otherwise - this script does not have to worry about that:

       ts="$(timestamp)"
       printf -- "%s" "$ts"  | pbench-add-metalog-option "${mdlog}" pbench timestamp

is all that's needed here. BTW, you want to use the `printf -- "%s" ....` idiom, so that we avoid troubles with % signs (see your other PRs :-) ).

Reply:
@ndokos I have tried this earlier but it is not giving any output when run.
```timestamp =                                 ```